tests/integration/conftest.py

```diff
@@ -53,3 +58,82 @@ def engine(
 @fixture(scope="session")
 def connection(engine: Engine) -> Connection:
     return engine.connect()
+
+
+@fixture(scope="session")
+def event_loop():
```

Comment:
Just curious, does this solve any issues with db api?

Reply:
What issues do you have in mind? 
This was added to extend the scope of event loop as it didn't play nice with the session-level fixtures for creating an engine/connection.

Reply:
Ok, I see. It's just that I've used such override to fix some async issues in the past before they were fixed, thought that might be the case